AutomatedTesting/Gem/PythonTests/editor/EditorScripts/AssetBrowser_SearchFiltering.py

```diff
@@ -122,7 +122,8 @@ def verify_files_appeared(model, allowed_asset_extensions, parent_index=QtCore.Q
         # 4) Click the "X" in the search bar.
         clear_search = asset_browser.findChild(QtWidgets.QToolButton, "ClearToolButton")
         clear_search.click()
-
+        
+        print("test1")
```

Comment:
Remove this debug statement and below as well

Reply:
Removed